tests/src/JIT/config/benchmark/benchmark.csproj

```diff
@@ -76,6 +76,9 @@
     <PackageReference Include="System.Threading">
       <Version>4.4.0-beta-24913-02</Version>
     </PackageReference>
+    <PackageReference Include="System.Threading.Thread">
```

Comment:
It looks like you removed the references to System.Threading in the tests.  Do you need this package reference?

Reply:
This is the error I get if I remove it. I don't know why it appeared all of the sudden, because my changes seem unrelated. Should I investigate this more or go with this change?

```
fannkuch-redux-5.cs(139,31): error CS0246: The type or namespace name 'Thread' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\vigraf\coreclr\tests\src\JIT\Performance\CodeQuality\BenchmarksGame\f annkuch-redux\fannkuch-redux-5.csproj]
fannkuch-redux-5.cs(143,35): error CS0246: The type or namespace name 'Thread' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\vigraf\coreclr\tests\src\JIT\Performance\CodeQuality\BenchmarksGame\f annkuch-redux\fannkuch-redux-5.csproj]
reverse-complement-6.cs(34,16): error CS0246: The type or namespace name 'Thread' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\vigraf\coreclr\tests\src\JIT\Performance\CodeQuality\BenchmarksGam e\reverse-complement\reverse-complement-6.csproj]
```

Reply:
Go ahead and leave it.  I don't think it hurts... I'm just surprised.